biz.aQute.bndlib/src/aQute/bnd/metatype/MetatypeAnnotations.java

```diff
@@ -28,11 +28,13 @@
 		nested
 	}
 
+	private static final Parameters	DEFAULT_HEADER	= OSGiHeader.parseHeader("*");
+
 	public boolean analyzeJar(Analyzer analyzer) throws Exception {
 		Map<TypeRef,OCDDef> classToOCDMap = new HashMap<TypeRef,OCDDef>();
 		Parameters header = OSGiHeader.parseHeader(analyzer.getProperty(Constants.METATYPE_ANNOTATIONS));
```

Comment:
Instead of the DEFAULT_HEADER, do `analyzer.getProperty(Constants.METATYPE_ANNOTATIONS, "*")`. 

header.size()==0 will match `-metatypeannotations: `.